instrumentation-loongsuite/loongsuite-instrumentation-mem0/pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "loongsuite-instrumentation-mem0"
+dynamic = ["version"]
+description = "LoongSuite Mem0 Instrumentation"
+readme = "README.md"
+license = "Apache-2.0"
+requires-python = ">=3.9"
+authors = [
+    { name = "LoongSuite Python Agent Authors", email = "sunyumei1991@gmail.com" },
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+]
+dependencies = [
+    "wrapt",
+    "opentelemetry-api",
+    "opentelemetry-instrumentation",
```

Comment:
You may add version restrictions for dependencies as per the requirements in your README.md.
<img width="464" height="141" alt="image" src="https://github.com/user-attachments/assets/4e0e2114-25c3-431d-8bfe-6f886f04583e" />